run_cart.sh

```diff
@@ -7,17 +7,17 @@
 #SBATCH -o stdout.o%j           # Name of stdout output file
 #SBATCH -e stderr.e%j           # Name of stderr error file
 #SBATCH -A STAR-Intel           # Project Name
-#SBATCH -p skx-dev              # Queue (partition) name
-#SBATCH -N 4                    # Total # of nodes
-#SBATCH -n 40                   # Total # of mpi tasks (48 x  Total # of nodes)
+#SBATCH -p skx-normal           # Queue (partition) name
+#SBATCH -N 10                   # Total # of nodes
+#SBATCH -n 100                  # Total # of mpi tasks (48 x  Total # of nodes)
```

Comment:
Minor: Seems like you are only requesting 10 tasks per node but in comment it says (48 x num of nodes).
Just wanted to bring it in your attention.